src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -429,7 +349,6 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         return { ...address, name };
     }
 
-    private _mapStripePaymentData(element: StripePaymentMethodType.CreditCard, shouldSaveInstrument: boolean): StripeConfirmCardPaymentData;
     private _mapStripePaymentData(element: StripePaymentMethodType.iDEAL, returnUrl?: string): StripeConfirmIdealPaymentData;
     private _mapStripePaymentData(element: StripePaymentMethodType.Sepa): StripeConfirmSepaPaymentData;
     private _mapStripePaymentData(element: StripePaymentMethodType, arg2?: any): StripeConfirmPaymentData {
```

Comment:
Sorry, I know this is not part of this PR but can we give this variable a meaningful name, and maybe give it a type?

Reply:
Totally agree.